ports/espressif/boards/autosportlabs_esp32_can_x2/mpconfigboard.mk

```diff
@@ -0,0 +1,14 @@
+USB_VID = 0x16D0
+USB_PID = 0x07F2
```

Comment:
This is a "reusable" VID/PID, but we would prefer that you get a unique one. You can apply for one from Espressif at https://github.com/espressif/usb-pids, or you can get one from https://pid.codes, assuming your design is open source.

Reply:
Hi @dhalbert , we actually purchased this PID from https://www.mcselec.com/index.php?page=shop.product_details&flypage=shop.flypage&product_id=92&category_id=20&option=com_phpshop&Itemid=1

As you can see, their VID was grandfathered in, and USB-ORG would never issue a VID for 0x16D0. 

0x07F2 is one of 4 PIDs we purchased from MCS.  I have confidence they won't re-issue these PIDs from something else. 

I'm happy to forward my order confirmation to you via email. 

With that, is it OK to proceed as is? 

Thank you!